lib/spaces/engines/packing/space.rb

```diff
@@ -21,88 +19,21 @@ def by(identifier, klass = default_model_class)
       end
     end
 
-    def encloses_commit?(descriptor); encloses_good_result?(:commit, descriptor) ;end
-    def encloses_export?(descriptor); encloses_good_result?(:export, descriptor) ;end
-
-    def encloses_good_result?(command, descriptor)
-      encloses_result?(command, descriptor) && with_good_artifacts?(command, descriptor)
-    end
-
-    def encloses_result?(command, descriptor)
-      path_for(descriptor).join(command.to_s).exist?
-    end
-
-    def with_good_artifacts?(command, descriptor)
-      artifacts_by(command, descriptor)&.any?(&:id)
-    end
-
-    def artifacts_by(command, descriptor)
-      YAML::load(path_for(descriptor).join("#{command}", 'artifacts.yaml').read)
-    rescue Errno::ENOENT => e
-      nil
-    end
-
     def save(model)
       raise PackWithoutImagesError, "Model doesn't have images: #{model.identifier}" unless model.has?(:builders)
 
       ensure_connections_exist_for(model)
       super.tap do
-        path_for(model).join('commit.json').write(model.to_h.to_json)
+        path_for(model).join('payload.json').write(model.to_h.to_json)
```

Comment:
Can you change this to `template.json` please as that's what packer calls it.

Reply:
yep

Reply:
change committed